### ISSUE TITLE
Permissions Issue

### ISSUE THREAD

**Issue by mattrothenberg** [spicywebau/craft-embedded-assets#92]
Howdy,

I just bumped Craft to 3.1.1, and noticed that I'm getting the following 401 when trying to create an embedded asset as a non-admin user.

![screen shot 2019-01-16 at 10 41 25 pm](https://user-images.githubusercontent.com/5148596/51294209-f3c9c980-19df-11e9-91c4-fbad9c54d4f4.png)

I can verify that the user has the correct permissions `saveAssetInVolume`, but the call continually fails.

**Update**
I noticed that `$folderId` evaluates to 0, although my `project.yaml` reflects the following permission: `saveassetinvolume:469523e8-795a-4edf-84ae-6d6e92694ca9`. Could this discrepancy in `folderId` be causing the issue? If so, how could these IDs have drifted apart from one another?

Any ideas?

**Comment by 24HOURSMEDIA** [comment]
I have this issue too, the way the folderId is obtained seems to fail.
The result of the save assets Ajax call gives:

```
{
    "success": true,
    "payload": {
        "assetId": 824,
        "folderId": "0"
    }
}
```

And the asset is always saved in folder id=1 I see in the database..

**Comment by mattrothenberg** [comment]
Switched to https://github.com/wrav/oembed/. @24HOURSMEDIA, you might want to give that a whirl!

**Comment by chrisrowe** [comment]
+1 on this issue

**Comment by chrisrowe** [comment]
@mattrothenberg Could you reopen this issue rather than have me duplicate it?

**Comment by chrisrowe** [comment]
I've moved to https://github.com/limesquare-nl/craft-embedded-assets which appears to fix this issue.

**Comment by ttempleton** [comment]
If @limesquare-nl submits a pull request with the fix for this, we'll be happy to merge it into the `dev` branch for release in the near future.

**Comment by ttempleton** [comment]
Update: the fix for this has been included in #84.  A new release will be out in the next couple of days and we'll keep this issue open until then.

**Comment by ttempleton** [comment]
Embedded Assets 2.0.0 has now been released with the fix for this.  With Embedded Assets now being maintained by Spicy Web, it should be updated from a terminal by running `composer require spicyweb/craft-embedded-assets` and then `composer remove benjamminf/craft-embedded-assets`.
